packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -258,22 +236,6 @@ class AndroidDevice extends Device {
     return <String>[_androidSdk.adbPath, '-s', id, ...args];
   }
 
-  String runAdbCheckedSync(
```

Comment:
Removed the sync call because for processes async is better, and it was only used once